pkg/controller/restapi/operation/messaging/models.go

```diff
@@ -0,0 +1,84 @@
+/*
+ *
+ * Copyright SecureKey Technologies Inc. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * /
+ *
+ */
+
+package messaging
```

Comment:
@sudeshrshetty @troyronda - just wondering if we need operation folder in between restapi and feature(messaging). These are already placed inside controller folder. ie. `pkg/controller/restapi/messaging` would make more sense.

Reply:
good point.

Reply:
Yes we need operation folder for now, rest api initializer, opts and other APIs from ‘pkg/restapi’ will move under ‘pkg/controller/restapi’.. we can revisit folder structure once I move all REST packages.